dev/create_release_tag.py

```diff
@@ -41,9 +41,16 @@
 def main(new_version: str, remote: str, dry_run: bool = False):
     version = Version(new_version)
     release_branch = f"branch-{version.major}.{version.minor}"
+    if dry_run:
+        subprocess.run(["git", "checkout", "-b", release_branch], check=True)
     release_tag = f"v{new_version}"
-    subprocess.check_call(["git", "tag", release_tag, release_branch])
```

Comment:
Could we just remove `release_branch`? So the role of this script is just create a specified release tag on the current branch?

Reply:
ah yes! Great idea. This runs on a fork of the repo anyway. Making adjustment!